tema5/src/main/java/ro/scoalainformala/Hobby.java

```diff
@@ -0,0 +1,36 @@
+package ro.scoalainformala;
+
+
+import java.util.List;
+
+public class Hobby {
+
+    private String nameOfHobby;
+    private int frequency;
+    List<AddressForHobby> addresses;
+
+    public Hobby(int frequency, String nameOfHobby, List<AddressForHobby> addresses) {
+        this.frequency = frequency;
+        this.nameOfHobby = nameOfHobby;
+        this.addresses = addresses;
+    }
+
+    public String getNameOfHobby()
+    {
```

Comment:
```suggestion
    public String getNameOfHobby() {
```

try to follow java convention